src/imgix-core-js.js

```diff
@@ -47,7 +47,7 @@
         this.settings.domains = [this.settings.domains];
       }
       else {
-        console.warn("Warning: Domain sharding has been deprecated and will be removed in the next major version.");
+        console.warn("Warning: Domain sharding has been deprecated and will be removed in the next major version.\nAs a result, the 'domains' argument will be deprecated in favor of 'domain' instead.");
```

Comment:
@jayeb I tagged you for review in case you think there's a better way to communicate this

Reply:
Naw, seems fine.